src/js/components/Card.js

```diff
@@ -193,8 +193,15 @@ export default class Card extends Component {
     let link = this._renderLink();
     let videoLayer = this._renderVideoLayer();
 
+    const contentClasses = classnames(
+      {
+        [`${CLASS_ROOT}__content`]: true,
+        [`${CLASS_ROOT}__content--truncated`]: truncate
```

Comment:
Can we align the class label with the property name and just call the class `__content--truncate`?